docs/docs/flink-writes.md

```diff
@@ -262,6 +262,107 @@ INSERT INTO tableName /*+ OPTIONS('upsert-enabled'='true') */
 
 Check out all the options here: [write-options](flink-configuration.md#write-options) 
 
+## Distribution mode
+
+Flink streaming writer supports both `HASH` and `RANGE` distribution mode.
+You can enable it via `FlinkSink#Builder#distributionMode(DistributionMode)`
+or via [write-options](flink-configuration.md#write-options).
+
+### Hash distribution
+
+HASH distribution shuffle data by partition key (partitioned table) or
+equality fields (non-partitioned table). It simply leverages Flink's
```

Comment:
Is this a technical detail which needed in the doc?
This is true, but has nothing to do with the configuration. It is a requirement for CDC writing

Reply:
Agree that it is a bit technical. I am thinking it might help explaining the limitations below.